Library/Homebrew/metafiles.rb

```diff
@@ -13,13 +13,23 @@ module Metafiles
 
   module_function
 
+  def license?(file)
+    file = file.upcase
+    ext = File.extname(file)
+    file = File.basename(file, ext)
+    file.gsub!("LICENCE", "LICENSE")
+    file == "LICENSE" || file.start_with?("LICENSE-")
+  end
+
   def list?(file)
     return false if %w[.DS_Store INSTALL_RECEIPT.json].include?(file)
 
     !copy?(file)
   end
 
   def copy?(file)
+    return true if license?(file)
```

Comment:
Now the two entries in `BASENAMES` are not used anymore.

Reply:
True that, do we want to get rid of them?

Reply:
Either that, or move them to something like `BASENAME_REGEXES`:

```ruby
BASENAME_REGEXES = [/^licen(c|s)e(\-|$)/].freeze
```

and change the check to

```ruby
BASENAMES.include?(file) || BASENAME_REGEXES.any? { |r| r.match?(file) }
```

Reply:
Doesn't even need to be an array of regexes.